README.md

```diff
@@ -124,13 +124,17 @@ git --version
    fprime-util build -j20
    ```
 
+1. Find your board's mount location
+   > For the next steps you'll need to know where your board is mounted. To find your board's mount location run  `ls -lah /dev/tyy*` for a list of all possible paths. On Linux and WSL your board's mount location will look like `/dev/ttyACM0`. On Mac your board's mount location will look like `/dev/tty.usbmodem101`.
```

Comment:
Ooh I think there is a small typo here: 

> `ls -lah /dev/tyy*`

Should be
> `ls -lah /dev/tty*`

Reply:
```suggestion
   > For the next steps you'll need to know where your board is mounted. To find your board's mount location run  `ls -lah /dev/tty*` for a list of all possible paths. On Linux and WSL your board's mount location will look like `/dev/ttyACM0`. On Mac your board's mount location will look like `/dev/tty.usbmodem101`.
```